catalog/conformance-tests/result-fujitsu-001.json

```diff
@@ -0,0 +1,14 @@
+{
+  "tested_solution": {
+    "solution_id": "fujitsu",
+    "version": "1.0.0"
+  },
+  "tested_by": {
+    "solution_name": "nri-cts",
+    "version": "0.9.15",
+    "solution_id": "NRI-CTS"
```

Comment:
@ta-Kuwayama -  I guess the solution_id and solution_name for NRI are swapped. please update them and it works hopefully.